tools/travis-ci/run.sh

```diff
@@ -303,4 +305,43 @@ New/unknown test failure found!!!
 	fi
 fi
 
+if [ "${OPJ_CI_PERF_TESTS:-}" == "1" ]; then
+    cd tests
+    echo "Running performance tests on current version (dry-run)"
+    PATH=../build/bin:$PATH python ./perf_test.py
+    echo "Running performance tests on current version"
+    PATH=../build/bin:$PATH python ./perf_test.py -o /tmp/new.csv
+    if [ "${OPJ_NONCOMMERCIAL:-}" == "1" ] && [ -d ../kdu ]; then
+        echo "Running performances tests with Kakadu"
+        LD_LIBRARY_PATH=../kdu PATH=../kdu::$PATH python ./perf_test.py -kakadu -o /tmp/kakadu.csv
+        echo "Comparing current version with Kakadu"
+        python compare_perfs.py /tmp/kakadu.csv /tmp/new.csv || true
+    fi
+    cd ..
+
+    REF_VERSION=master
+    if [ "${TRAVIS_PULL_REQUEST:-false}" == "false" ]; then
+        REF_VERSION=v2.1.2
+    fi
+    if [ ! -d ref_ojp ]; then
```

Comment:
strangely enough, the correct prefix or suffix for OpenJPEG is opj and not ojp ... As everything in the code follows opj, I suggest we keep this rule.